parking/backend/sensor_server/rest_server.py

```diff
@@ -0,0 +1,81 @@
+import json
+from tornado import web
+from parking.shared.rest_models import (ParkingLot, ParkingLotCreationResponse, ParkingLotAvailableMessage,
+                                        ParkingLotPriceMessage, InvalidLotFieldsError, InvalidLotIDError,
+                                        InvalidLotAvailableError, RESTError, InvalidLotPriceError)
+from parking.shared.util import serialize_model
+from parking.backend.db.dbaccess import DbAccess
+
+
+class ParkingLotsHandler(web.RequestHandler):
+    def initialize(self, dba: DbAccess) -> None:
+        self.dba = dba
+
+    async def post(self):
+        try:
+            lot = ParkingLot(**json.loads(self.request.body))
+        except (TypeError, ValueError) as err:
+            self.write(serialize_model(self._create_error_response(err)))
```

Comment:
We could probably let this propagate out as an HTTP error response.
I'd suggest defining an alternate constructor like `ParkingLot.from_json` that raises [`HTTPError`](http://www.tornadoweb.org/en/stable/web.html#tornado.web.HTTPError)

We could then wrap this try-except up as a utility function:

    def try_load(cls, json_payload):
         try:
             return cls(**json.loads(json_payload))
         except (TypeError, ValueError) as err:
             raise HTTPError(400, log_message="Bad Request ...")

Then we don't have to actually manually create responses for malformed requests.